go/store/datas/commit_options.go

```diff
@@ -28,9 +28,9 @@ import (
 
 // CommitOptions is used to pass options into Commit.
 type CommitOptions struct {
-	// Parents, if provided is the parent commits of the commit we are
+	// ParentsList, if provided is the parent commits of the commit we are
 	// creating.
-	Parents types.Set
+	ParentsList types.List
```

Comment:
Or leave as `Parents` now that the conversion is done?

Reply:
ParentsList is good.